src/plugin.js

```diff
@@ -334,7 +349,7 @@ const eme = function(options = {}) {
           if (event.type === 'mskeyerror') {
             callback(event.target.error);
             if (!suppressErrorIfPossible) {
-              emeError(event.message);
+              emeError(event.message, videojs.Error.MSKeyError);
```

Comment:
as we discussed offline:
we can analyze each rejection here, and try to use spec errors for legacy flow